bus/src/main/java/org/hawkular/metrics/component/publish/AvailDataMessage.java

```diff
@@ -104,6 +104,8 @@ public String toString() {
         @JsonInclude
         private String tenantId;
         @JsonInclude
+        private String type;
```

Comment:
Ok, thanks for the feedback, it seems it creates more complexity.
I am reverting this one.